spec/api-app-spec.js

```diff
@@ -862,13 +861,16 @@ describe('app module', () => {
       }
     })
 
-    it('fails for invalid info_type', () => {
+    it('fails for invalid info_type', async () => {
       const invalidType = 'invalid'
       const errorMessage =
           `app.getGPUInfo() didn't fail for the "${invalidType}" info type`
-      return app.getGPUInfo(invalidType).then(
-        () => Promise.reject(new Error(errorMessage)),
-        () => Promise.resolve())
+      try {
+        await app.getGPUInfo(invalidType)
+        Promise.resolve()
```

Comment:
If we're expecting this to fail I'd imagine this code should more look like

```js
try {
  await app.getGPUInfo(invalidType)
  throw new Error(errorMessage)
} catch (err) {
  // We are expecting an error here
}
```

but as a super improvement to this spec, you can actually just write it like this

```js
return expect(app.getGPUInfo(invalidType)).to.eventually.be.rejectedWith(/expected error/)
```